styles.css

```diff
@@ -0,0 +1,90 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;1,100;1,200&display=swap");
+
+:root {
+  --primary-blue: #6070ff;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background: url("./img/header-shapes\ mobile@2x.svg");
+  background-position: 0 3.75rem;
+  background-repeat: repeat-x;
+  background-size: cover;
+}
```

Comment:
# Required Changes ♻️ 

- [ ] The background image is supposed to be on the headline section and not the body of the website. Please fix that by placing the background image on the headline section.
- [ ] It would be nice you match your design to the one given in Figma
- The Figma Design
![image](https://user-images.githubusercontent.com/50941074/158701436-04c1ca80-d2f1-4ca2-80f4-d4aa25bd8439.png)

- Your Design
![image](https://user-images.githubusercontent.com/50941074/158701511-64ad5481-e0c6-40c8-b2c5-5df8017ced85.png)